src/components/nav-user/nav-user.tsx

```diff
@@ -0,0 +1,111 @@
+import { Component, Element, h, Host, Prop, Event, EventEmitter } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-nav-user",
+  styleUrl: "nav-user.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavUser {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteNavUserElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When `true`, visually highlight the component */
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the text to display, for example a user name or full name */
+  @Prop() fullName: string;
+
+  /** Specifies accesible label for the component */
+  @Prop() label: string;
```

Comment:
doc consistency recommendation - if needed, depends on https://github.com/Esri/calcite-components/issues/5563 and https://github.com/Esri/calcite-components/issues/5564, perhaps?:

> Specifies the label of the component.